Assets/Mona/QualityAssurance/QualityAssurance.ErrorDescriptions.cs

```diff
@@ -0,0 +1,35 @@
+public static class ErrorDescriptions
```

Comment:
I think we could put this in the same file as the error codes so it's easier to manage. `GetErrorDescriptionMap` is pretty tightly-coupled with both of those classes so it's a little anti-pattern to leave one of them in a separate file.

Reply:
You could also probably omit the `_DESCRIPTION` at the end of each variable name since it's redundant now that they're all in a Descriptions struct